src/tests/unit/test-data.js

```diff
@@ -452,6 +452,211 @@ const ingredientSerializerData = {
   },
 };
 
+const serializePizzaData = {
+  baseInputPizza: {
+    id: '1',
+    name: 'sample pizza',
+    ovenTemp: '500',
+    bakeTime: '10',
+    specialInstructions: null,
+    ingredients: [
+      {
+        id: '1',
+        name: 'sausage',
+        ingredientType: 'meat',
+        notes: null,
+      },
+    ],
+    dough: {
+      name: 'weeknight pizza dough',
+      id: '201',
+      gramsFlour: '500',
+      flourType: 'All Purpose',
+      gramsWater: '400',
+      waterTemp: '90',
+      gramsYeast: '5',
+      gramsSalt: '15',
+      bulkFermentTime: '60',
+      proofTime: '15',
+      gramsSugar: '0',
+      gramsOliveOil: '0',
+      specialInstructions: null,
+    },
+  },
+  goalSerializedPizza: {
+    data: {
+      attributes: {
+        name: 'sample pizza',
+        ovenTemp: 500,
+        bakeTime: 10,
+        specialInstructions: '',
+      },
+      id: '1',
+      type: 'string',
+      relationships: {
+        ingredients: {
+          data: [
+            {
+              type: 'ingredient',
+              id: '1',
+            },
+          ],
+          links: {
+            related: '/v1/pizzas/1/ingredients',
+            self: '/v1/pizzas/1/relationships/ingredients',
+          },
+        },
+        dough: {
+          data: {
+            type: 'dough',
+            id: '201',
+          },
+          links: {
+            related: '/v1/pizzas/1/dough',
+            self: '/v1/pizzas/1/relationships/dough',
+          },
+        },
+      },
+    },
+    links: {
+      self: '/v1/pizzas/1',
+    },
+    included: [
+      {
+        type: 'dough',
+        id: '201',
+        attributes: {
+          name: 'weeknight pizza dough',
+          gramsFlour: 500,
+          flourType: 'All Purpose',
+          gramsWater: 400,
+          waterTemp: 90,
+          gramsYeast: 5,
+          gramsSalt: 15,
+          gramsSugar: 0,
+          gramsOliveOil: 0,
+          bulkFermentTime: 60,
+          proofTime: 15,
+          specialInstructions: '',
+        },
+        relationships: {},
```

Comment:
based on [this](https://github.com/SeyZ/jsonapi-serializer/pull/209) PR, I think the fact that `relationships` shows up here is a bug in the serializer. It seems mostly benign though, since it doesn't show up in the API output.

Reply:
You have to be real careful with odd bugs like this. They can create wonky edge cases that don't show up until a user finds it. At the very least you should understand why this is happening to actually determine its benignity.

Reply:
That's fair, I was kind of at a loss about what to do about since it was showing up really consistently in my tests but wasn't causing issues in the actual API. I'm not sure what the best next steps would be to verify it won't cause issues down the road.

Reply:
`relationships: {},` this actually does show up in the API output (won't show for the first call). @jemisonf can you look into it and fix it?